examples/EnTT/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2015, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-EnTT)
+
+# DOCUMENTATION_START {
+# EnTT requires at minimum C++17
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED on)
+
+hunter_add_package(EnTT)
+find_package(EnTT CONFIG REQUIRED)
+
+add_executable(foo foo.cpp)
+
+target_include_directories(foo
```

Comment:
Should it be imported target like `EnTT::EnTT`?

Reply:
Sadly no, I tried various incantations there but the only thing that works was `${ENTT_INCLUDE_DIRS}`, is this a bug in EnTT's build script?  Does @skypjack need to do something and make another release (maybe of a fixed 2.7.y line instead)?

Reply:
> is this a bug in EnTT's build script? Does @skypjack need to do something and make another release (maybe of a fixed 2.7.y line instead)?

Imported target is a best practice, note that if `ENTT_INLUDE_DIRS` is a full path then package is not relocatable.

> And they do end up changing toolchain settings, should EnTT remove that from the CMakeLists.txt and advise to use an appropriate toolchain instead if they want such things set?

Toolchain is a requirement for Hunter, not for CMake in general. The logic is next: if you want to build with libc++ then you have to specify libc++ for all packages in set. It's easier to create toolchain with libc++ than create option for all packages. I don't know what `USE_COMPILE_OPTIONS` do, but I guess you can just add `USE_LIBCPP=OFF` to default `CMAKE_ARGS` and everything will be fine.